class.jetpack.php

```diff
@@ -3930,10 +3930,11 @@ function admin_page_load() {
 			Jetpack::restate();
 		}
 
+		$from = ! empty( $_GET['from'] ) ? $_GET['from'] : 'iframe';
```

Comment:
Should we move this just inside `if ( isset( $_GET['connect_url_redirect'] ) ) {` below? Seems like it's not necessary outside.